applications/GeoMechanicsApplication/tests/lysmer_boundary_column3D/README.md

```diff
@@ -0,0 +1,48 @@
+# Lysmer absorbing boundary conditions on a column made of 3D elements
+
+These 3D tests verify the P-wave propagation through 1D column.  
```

Comment:
To me '1D' is a bit confusing (also in line 7), since the column itself is 3D (1x1x10) as you describe in the rest of the documentation

Reply:
Put some words on it. Indeed we test the 3D code using the 1D problem. It means blocking any movements in other two directions.